### DISCUSSION TITLE
A small, medium and large organization can create my own badge for your contributors?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31526] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everybody! 

Recently GitHub adopted the Achievements feature, which allows the developer to earn achievement badges. A doubt: 
1. A small, medium and large organization can create my own badge for your contributors?
2. Is it possible to create my own badge on GitHub?

**Comment by jfullstackdev** [comment]
as far as I know, there is none

there are external services that can create a badge, but they are not considered official since you cannot put that at your profile badges but in the default markdown of your landing page

**Reply by ghost** [reply]
@jdevstatic How can I request this feature?

**Reply by jfullstackdev** [reply]
yes, you can always make that request, but GitHub will prioritize everything that they think are essential 

as for now, you can look for external services that can create a badge that you can embed on your markdown, like the GitHub Stats that I'm using that can increase the pins for the README

**Reply by ralphleyga67625520** [reply]
wow

**Comment by melvincwng** [comment, marked as answer]
@codehangen, regarding your two questions above:

1. Unfortunately, no - there's no functionality for an organization to create custom badges for their members.

2. No as well - you can't create your own badges on GitHub. The GitHub team decides what badges or achievements they want on their platform.

I saw your comment as well regarding the requesting of such features. With regards to this, there isn't any official way to request such features (as far as I'm aware of).

Usually, for feature requests, I notice members of the community create discussion threads (like this one), and once it gains enough traction, usually a GitHub staff will pop in and notify if the feature will be implemented in future.

Take for instance, in https://github.com/orgs/community/discussions/31644, the Trending Tab was going to be deprecated in the future version of GitHub. Many members of the community wanted this feature to remain, and as you can see in the 'Marked answer' comment, a GitHub staff stepped in to mention that they have noted the comments and hence need to relook at "what the long term plan is for the Trending page".

In short, for your feature request to be successful, it probably has to gain sufficient traction by the community.

Hope this answers your question 😊

**Reply by ghost** [reply]
@melvincwng hey there! thank you for feedback.

**Reply by melvincwng** [reply]
@codehangen always glad to help 👍🏻

**Reply by Harish-2003** [reply]
i think its not true ? you can see gssoc an organizatino which assigns badges based upon their contributions
